5_databases/week1/homework.sql

```diff
@@ -0,0 +1,53 @@
+--> 1. Find out how many tasks are in the task table
+SELECT COUNT(*) AS hom_many_tasks
+FROM task;
+
+--> 2. Find out how many tasks in the task table do not have a valid due date
+SELECT  COUNT(*) AS task_no_due_data
+FROM task
+WHERE due_date IS NULL
+
+--> 3. Find all the tasks that are marked as done
+SELECT task.*
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+--> 4. Find all the tasks that are not marked as done
+SELECT task.*
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name != 'Done';
+
+
+--> 5. Get all the tasks, sorted with the most recently created first
+SELECT * FROM task ORDER BY created DESC
+
+--> 6. Get the single most recently created task
+SELECT *
+FROM Task
+ORDER BY created DESC
+LIMIT 1;
+
+--> 7. Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date
+FROM task
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+
+--> 8. Get the title and status (as text) of all tasks
+SELECT task.title, status.name AS status_text
+FROM task
+JOIN status on task.status_id = status_id
+
+--> 9. Get the name of each status, along with a count of how many tasks have that status
+SELECT status.name AS status_name, COUNT(*) AS task_count
```

Comment:
This is the same problem as above, it returns the tasks 3 times. The current output is:


```
+-------------+----------------+
| name        | COUNT(task.id) |
+-------------+----------------+
| Done        |             35 |
| In progress |             35 |
| Not started |             35 |
+-------------+----------------+
```
If you fix the SQL statement above this should be fixed as well.

Reply:
Are your sure? Because I have this output https://skr.sh/sMrEcTGaNAp

Reply:
But I made the changes anyway :)

Reply:
@ayaroshenkocom Ah, this was actually correct, sorry. Good job. The task above this one has to be fixed tho :)

Reply:
Ok. Thank you very much for the review :)